redi-connect-front/src/pages/app/me/Me.tsx

```diff
@@ -29,187 +41,163 @@ export type SignUpFormType =
-  githubProfileUrl: string
-  slackUsername: string
-  telephoneNumber: string
-  categories: string[]
-  menteeCountCapacity: number
 }
 
 const validationSchema = Yup.object({
-  mentor_occupation: Yup.string().when('formType', {
-    is: 'mentor',
-    then: Yup.string()
-      .required()
-      .max(255)
-      .label('Occupation')
-  }),
-  mentor_workPlace: Yup.string()
-    .max(255)
-    .label('Work place'),
-  mentee_occupationCategoryId: Yup.string().when('formType', {
-    is: 'mentee',
-    then: Yup.string()
-      .required()
-      .oneOf(menteeOccupationCategories.map(v => v.id))
-      .label('Current occupation')
-  }),
-  mentee_occupationJob_placeOfEmployment: Yup.string()
-    .max(255)
-    .label('Where are you employed'),
-  mentee_occupationJob_position: Yup.string()
-    .max(255)
-    .label('At what university do you study'),
-  mentee_occupationStudent_studyPlace: Yup.string()
-    .max(255)
-    .label('Where do you study'),
-  mentee_occupationStudent_studyName: Yup.string()
-    .max(255)
-    .label('What do you study'),
-  mentee_occupationLookingForJob_what: Yup.string()
-    .max(255)
-    .label('What kind of job'),
-  mentee_occupationOther_description: Yup.string()
-    .max(255)
-    .label('What are you currently doing'),
-  mentee_highestEducationLevel: Yup.string().when('formType', {
-    is: 'mentee',
-    then: Yup.string()
-      .oneOf(educationLevels.map(level => level.id))
-      .label('Highest Education Level')
-  }),
-  mentee_currentlyEnrolledInCourse: Yup.string().when('formType', {
-    is: 'public-sign-up-mentee-pending-review',
-    then: Yup.string()
-      .required()
-      .oneOf(courses.map(level => level.id))
-      .label('Currently enrolled in course')
-  }),
-  profileAvatarImageS3Key: Yup.string().max(255),
-  firstName: Yup.string()
-    .required()
-    .max(255),
-  lastName: Yup.string()
-    .required()
-    .max(255),
-  gender: Yup.string()
-    .oneOf(['male', 'female', 'other'])
-    .label('Gender'),
-  age: Yup.number()
-    .min(16)
-    .max(99)
-    .label('Age'),
-  languages: Yup.array()
-    .min(1)
-    .of(Yup.string().max(255))
-    .label('Languages'),
-  personalDescription: Yup.string()
-    .required()
-    .min(100)
-    .max(600)
-    .label('Personal description'),
-  contactEmail: Yup.string()
-    .email()
-    .required()
-    .max(255)
-    .label('Contact email'),
-  linkedInProfileUrl: Yup.string()
-    .max(255)
-    .url()
-    .label('LinkedIn Profile'),
-  githubProfileUrl: Yup.string()
-    .max(255)
-    .url()
-    .label('Github Profile'),
-  slackUsername: Yup.string()
-    .max(255)
-    .label('Slack username'),
-  telephoneNumber: Yup.string()
-    .max(255)
-    .label('Telephone number'),
-  categories: Yup.array().when('formType', {
-    is: 'mentee',
-    then: Yup.array()
-      .compact(v => v === 'dontKnowYet')
-      .min(0)
-      .max(3)
-  }),
-  menteeCountCapacity: Yup.number().when('formType', {
```

Comment:
Here the rule for the capacity

Reply:
Ah, then it's all good. Must mean that this is non-inclusive. So 0 and 3 actually becomes 1 and 2. Perhaps worth checking the yup library to see if we can flag these numbers as inclusive. something like `min(1, true) and max(2, true)`

Reply:
I think you are reading the wrong rule (categories). The one below is the one for the **menteeCountCapacity**
This is already inclusive so this would mean he can take from 1 to 4 mentees which would be incorrect if we need want to allow 1 or 2. The exclusive version would be with **lessThan** and **moreThan** according to the docu. 

I will update to be correct then

Reply:
Ah, my bad.

I'm a bit lost now - let's just make sure to implement these rules:
- for mentee count, it can be 1-2
- for category count, it can be maximum three

Reply:
Yup, this is how it is atm

Reply:
You're ReDI!